rllib/evaluation/rollout_worker.py

```diff
@@ -2108,6 +2109,16 @@ def _build_policy_map(
             else:
                 new_policy = policy
 
+            if self.config.get("_enable_rl_module_api", False):
```

Comment:
```
if self.config.get("_enable_rl_module_api", False):
    rl_module = getattr(new_policy, "model", None)
    if rl_module is not None and self.config.framework_str == "torch":
             if self.config.get("torch_compile_rollout_worker"):
                      compile_config = self.config.get_compile_worker_config()
                      rl_module = rl_module.compile(compile_config)
```

Reply:
you can modify the nested condition checking to make it less complex / prettier.